homeassistant/components/islamic_prayer_times/__init__.py

```diff
@@ -29,6 +29,14 @@ def update_unique_id(
 
     await er.async_migrate_entries(hass, config_entry.entry_id, update_unique_id)
 
+    # add lat and lon to entry data if not present
+    if not config_entry.data:
+        data = {
+            CONF_LATITUDE: hass.config.latitude,
+            CONF_LONGITUDE: hass.config.longitude,
+        }
+        hass.config_entries.async_update_entry(config_entry, data=data)
```

Comment:
In 2024.1, minor versions are added, I think this would be a good example to add it to. Let me double check with someone if this is indeed a good example.

Reply:
Any updates on this please.

Reply:
They are still going to write a blogpost, but I just saw an example PR fly by: https://github.com/home-assistant/core/pull/106109

Reply:
Documentation about minor version is available under https://developers.home-assistant.io/docs/config_entries_config_flow_handler#config-entry-migration